### PR TITLE
Add unblocked_terms.txt

### PR DESCRIPTION
**Pull request by Fabian-Sommer**
This CL enables a presubmit preventing future use of non-inclusive
language.
It also adds documentation on how to remove non-inclusive terms
currently used in this repository.

### CODE REVIEW

**Review by emaxx-google — APPROVED** [review]
Thanks! LGTM, but a question for my understanding - where exactly is this presubmit configured? Is this some default for all Google repositories, or something that's turned on only for this repo somehow?

**Comment by Fabian-Sommer** [comment]
Reading a bit more, I actually think the check is not enabled for github repos (even though the original email mentioned "all git repos"). I just went ahead and removed all offensive terms (exception: "master" which would require renaming the branch)

File: common/cpp/src/google_smart_card_common/logging/logging.h [hunk 1]
```diff
@@ -155,7 +155,7 @@ std::string MakeNotreachedHitMessage(const std::string& file,
 #define GOOGLE_SMART_CARD_NOTREACHED                                \
   do {                                                              \
     GOOGLE_SMART_CARD_LOG_FATAL                                     \
-        << ::google_smart_card::internal::MakeNotreachedHitMessage( \
+        << ::google_smart_card::internal::MakeNotreachedMessage(    \
```
Comment (emaxx-google) [inline common/cpp/src/google_smart_card_common/logging/logging.h:158]: nit: Check whether clang-format reformats this code by removing extra spaces till the backslash.

**Review by emaxx-google — APPROVED** [review]

**Review by ayaNader — APPROVED** [review]
LGTM

**Pull request merged**